networking_cisco/plugins/cisco/db/l3/l3_router_appliance_db.py

```diff
@@ -179,9 +179,11 @@ def _cisco_router_result_filter_hook(self, query, filters):
     def do_create_router(self, context, router, router_type_id, auto_schedule,
                          share_host, hosting_device_id=None, role=None,
                          inflated_slot_need=0):
+        r = router['router']
+        gw_info = r.pop(EXTERNAL_GW_INFO, None)
```

Comment:
what's the point of moving router_created section from the "with" block to outside?